loopy/target/c/__init__.py

```diff
@@ -692,7 +692,14 @@ def with_types(self, arg_id_to_dtype, callables_table):
                 raise LoopyTypeError(f"'{name}' needs order to be an int-type.")
 
             if arg_id_to_dtype[1].numpy_dtype == np.float32:
-                name_in_target = name[-2:]+"f"
+                # See e.g.
+                # https://github.com/flang-compiler/flang/blob/master/runtime/libpgmath/lib/common/mthdecls.h
```

Comment:
Not a stable (versioned) link. Also, I'm not totally clear what this might have to do with flang?

Reply:
> Not a stable (versioned) link.

I changed this to a stable link and added a Mac link in https://github.com/inducer/loopy/pull/702/commits/8c0d7a82b96abf5f4a73ef836dffe51f418147ca

> Also, I'm not totally clear what this might have to do with flang?

As far as I know, this only depends on the OS, not on the compiler. I just found the flang link more comprehensive than others that I found.